src/main/java/com/lob/net/APIResource.java

```diff
@@ -23,9 +23,15 @@
     }
 
     public static <T> LobResponse<T> request(APIResource.RequestMethod method, APIResource.RequestType type,
```

Comment:
We should get into the habit of writing javadoc for all our public methods.

Reply:
We can defer this to a later initiative to add javadoc to all public methods.